test/appium/tests/conftest.py

```diff
@@ -160,6 +205,28 @@ def pytest_runtest_makereport(item, call):
             current_test.testruns[-1].error = error
         if is_sauce_env:
             update_sauce_jobs(current_test.name, current_test.testruns[-1].jobs, report.passed)
+        if pytest.config.getoption('docker'):
```

Comment:
As you may see the [given hook](https://docs.pytest.org/en/latest/reference.html#_pytest.hookspec.pytest_runtest_makereport) has a special variable in its signature `item`. That's the proper way to get to pytest config object like [in the line](https://github.com/status-im/status-react/pull/6692/files#diff-01b737e8651719ddae14106d980b587fR198)

So, I propose you to change your code a bit within the hook according to the pytest doc.
Here what I mean:

Instead of
```python
def pytest_runtest_makereport(item, call):
    pytest.config.getoption('stats_db_host')
```
Do this
```python
def pytest_runtest_makereport(item, call):
    item.config.getoption('stats_db_host')
```

P.S. I know some pieces of pytest doc are not obvious and not well-documented but if you need some help here just ask me :)

Reply:
👍